mentoris/views.py

```diff
@@ -181,15 +216,6 @@ def latex(request):
 
             return redirect(f"../main/{volume_id}/{chapter_id}")
 
-        if "question-button" in request.POST:
-            answer = hidden_answer
-            grading = hidden_grading
-        if "answer-button" in request.POST:
-            question = hidden_question
-            grading = hidden_grading
-        if "grading-button" in request.POST:
-            question = hidden_question
-            answer = hidden_answer
```

Comment:
Double-check we can delete this stuff before merging. This helped make sure the input windows persisted after rendering.

Reply:
Yeah, that is no longer needed since we are using AJAX and each time a quiz is rendered the changes get saved to the server